nlp/train.py

```diff
@@ -0,0 +1,62 @@
+from pathlib import Path
+import pickle
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.naive_bayes import MultinomialNB
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import classification_report
+from sklearn.externals import joblib
+
+def main():
+    """Train model using tweet data"""
+    if (Path('nlp/tweets')).is_file():
+        filename = 'nlp/tweets'
+    else:
+        filename = 'tweets'
+
+    with open(filename, 'rb') as f:
+        tweets = pickle.load(f)
+
+        x = [] # data (tweets)
+        y = [] # category (emojis)
+
+        # Prep data
+        for key, value in tweets.items():
+            count = 0
+            print(key)
+            print(len(value))
+
+            for tweet in value:
+                count = count + 1
+                # Ensure the training data is the same number of entries so it's not biased
+                if (count < 4000):
```

Comment:
we can up this count once we get more data, but right now tweets file only contains data for 😮 and 😍 and i only have just under 4000 for 😮